romochka_bembot.js

```diff
@@ -56,11 +59,12 @@ const generateBemhtmlLink = bemhtml => {
  * @returns {String} Link
  */
 const generateBemjsonLink = bemjson => {
-	const trimedBemjson = _.trimEnd(bemjson, ';');
+	assert(typeof bemjson === 'string', 'Bemjson must be a string');
+
+	const trimedBemjson = bemjson.replace(/;+$/, '');
```

Comment:
Regular expression more difficult to understand and they are slower. Why it bad to use ladash? Looks like premature optimization